plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -680,6 +681,37 @@ public void testOptimizingWholeTableRemovesDeleteFiles()
         }
     }
 
+    @Test
+    void testRemoveOrphanDeletionVectors()
+            throws Exception
+    {
+        Session singleWriterPerTask = Session.builder(getSession())
+                .setSystemProperty("task_min_writer_count", "1")
+                .build();
+
+        Session shortRetentionUnlocked = Session.builder(getSession())
+                .setCatalogSessionProperty("iceberg", "expire_snapshots_min_retention", "0s")
+                .setCatalogSessionProperty("iceberg", "remove_orphan_files_min_retention", "0s")
+                .build();
+
+        try (TestTable table = newTrinoTable("expire_snapshots_dv", "(x int) WITH (format_version = 3)", List.of("1", "2"))) {
```

Comment:
why did we add this test in TestIcebergV2 file?

Reply:
We use this test class to verify Iceberg spec. Not only v2 but also v1 are tested. 
Let's rename it in a follow-up.  https://github.com/trinodb/trino/pull/24882#discussion_r1982284845